server/src/user/route.js

```diff
@@ -6,4 +6,5 @@ module.exports = (app) => {
   app.get('/api/users/:id', service.findById)
   app.put('/api/users/:id', service.update)
   app.delete('/api/users/:id', service.delete)
+  app.get('/auth/:token', service.activate)
```

Comment:
Move this to auth please.

Reply:
I think it is  related to user activation not to authentication, maybe i have to change the route uri name

Reply:
Then why auth? change it to `users/activation/:token`

Reply:
yes sure